packages/design-system/src/styles/components/_Table.scss

```diff
@@ -40,10 +46,19 @@
   }
 }
 
-.ds-c-table__header--label span {
-  display: block;
-  text-align: center;
-  text-transform: uppercase;
+@mixin ds-c-table--striped-style {
```

Comment:
Same here, why are you using a mixin? I see it being applied twice in this style sheet but I dont think thats necessary. Generally we dont add mixins unless they are intended to be used across files or by the user. they are also supposed to be defined in our mixins folder. That's why I dont think we should be defining mixins without a specific purpose

Reply:
what do you think if i change it to extend for %ds-c-table--striped-style as it's called in 2 places?

Reply:
Ah I forgot that makes sense. If thats the case could we name it something less official? like `striped-table`